backend/inspirehep/celery.py

```diff
@@ -34,6 +35,8 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
                     term_log.write(str(exc))
             finally:
                 os.kill(os.getppid(), signal.SIGTERM)
+        else:
+            Reject(requeue=True)
```

Comment:
Could you please add a test for this?

Reply:
Sure. Are there any tests I could use as a reference? I haven't done any tests for celery yet, so I'm not sure where to create one.

Reply:
wouldn't you need to `raise` this, as it's an exception? :thinking:

Reply:
That is my understanding also from the documentation, but when I raise it I get this error from several tests ```celery.exceptions.Reject: (AttributeError("'NoneType' object has no attribute 'create_ticket'"), True)```